.github/workflows/deploy.yml

```diff
@@ -17,9 +16,7 @@ jobs:
       postgres_unit_testing: true
       db_name: fsd_app_store_test
       deploy_to_dev: false
```

Comment:
In account store we now set to True. Are we okay setting this to True for Application store @gio-karunakaran?